### PR TITLE
styling/registration

### PR DESCRIPTION
**Pull request by tombjarne**
Added navigation/pagination for registration process

Minor style changes in skills and finish screen

registration

### CODE REVIEW

File: src/components/pages/registration/skills.js [hunk 1]
```diff
@@ -19,36 +22,30 @@ class RegistrationSkills extends React.Component {
...
                 <Form.Check.Label>Will you travel by bicycle or walk ?</Form.Check.Label>
                 <Form.Check.Input type="checkbox" name="walker"/>
               </Form.Check>
-              <Form.Check>
+              <Form.Check className="checkbox-wrapper">
                 <Form.Check.Label>Can you help with home work ?</Form.Check.Label>
                 <Form.Check.Input type="checkbox" name="artist"/>
               </Form.Check>
-              <Form.Check>
+              <Form.Check className="checkbox-wrapper">
```
Comment (santiagoarriaga) [inline src/components/pages/registration/skills.js:45]: @tombjarne when I display the page, I don't see the new styling on this page. Am I missing something ?

Comment (tombjarne) [reply src/components/pages/registration/skills.js:45]: @santiagoarriaga you might need to install the dependencies for the Pagination in React Bootstrap. It actually should have updated the package.json as well with the new dependency.

Comment (santiagoarriaga) [reply src/components/pages/registration/skills.js:45]: @tombjarne also, the step styling looks much uglier in my browser than in the capture you had, are some styles missing in the branch ?

Comment (tombjarne) [reply src/components/pages/registration/skills.js:45]: @santiagoarriaga Ah right! The App.css changes were not taken over. I just pushed it to the current branch! Sorry I missed out on that one

Comment (santiagoarriaga) [reply src/components/pages/registration/skills.js:45]: @tombjarne with latest update now looks good, thanks !

**Pull request merged**